frappe/app.py

```diff
@@ -388,7 +388,7 @@ def handle_exception(e):
 		if hasattr(frappe.local, "login_manager"):
 			frappe.local.login_manager.clear_cookies()
 
-	if http_status_code >= 500:
+	if http_status_code >= 400 and http_status_code not in [401, 403, 404]:
```

Comment:
Can we also exclude 417? those are validation error `frappe.throw`

Reply:
I thought about that and they could be very useful for debugging customer problems because there are so many of them. So I think they should be included. Especially since the defer mechanism is in place and the error log table is pruned, there's no real harm in keeping them.